cache/disk/options.go

```diff
@@ -56,3 +59,36 @@ func WithAccessLogger(logger *log.Logger) Option {
 		return nil
 	}
 }
+
+func WithEndpointMetrics() Option {
+	return func(c *Cache) error {
+		c.acHit = promauto.NewCounter(prometheus.CounterOpts{
```

Comment:
These could be passed in as arguments, and the disk module would no longer depend on the prometheus library.